src/index.scss

```diff
@@ -265,7 +265,10 @@ $collapseHoverColor: #353535;
 .glimpse-ajax-row {
     white-space: nowrap !important;
     position: relative !important;
-    animation: glimpse-ajax-row-enter .3s ease-out;
+    animation: glimpse-ajax-row-enter .3s ease-out !important;
+    border: none !important;
+    padding: 0 !important;
+    margin: 0 !important;
```

Comment:
I think this should be fine unless git branches hadn't messed up, this is my main concern.